scripts/export_prefix_check.sh.in

```diff
@@ -50,6 +57,16 @@ exports=$(find @CMAKE_BINARY_DIR@ -not -path '*/\.*' -name '*.so' -print0 \
         -e '^_\?timescaledb_' \
         -e '^_\?loader_hello$' \
         -e '^_\?test_symbol_conflict$' \
+        -e '^_\?__sanitizer' \
+        -e '^_\?__sancov' \
+        -e '^_\?__asan' \
+        -e '^_\?__odr_asan' \
+        -e '^_\?__ubsan' \
+        -e '^_\?_ZN7__ubsan' \
+        -e '^_\?_ZN11__sanitizer' \
+        -e '^_\?internal_sigreturn$' \
+        -e '^_\?OnPrint$' \
+        -e '^_\?backtrace_uncompress_zdebug$' \
```

Comment:
we wouldn't need this if we don't run the export_prefix_check in this workflow

Reply:
True, but should we keep it though?

Reply:
no strong oppinion either way

Reply:
I suggest simply skipping export_prefix_check in this workflow

Reply:
The `export_prefix_check` seems to be there only to check that we are prefixing exported symbols with `ts_` (see commit bb6efbef0) and as such I do not think you need it for the sanitizer run. I note that we have a tendency to jam  checks like this into workflows rather than defining separate workflows for doing more focused checks.

I also think you can remove that step in the workflow as long as it exists in some other workflow; and it is already present in the `regress` workflow.

Reply:
`export_prefix_check.sh` has already been removed from this workflow. The remaining question is if I should remove the above symbols as well.

Reply:
Since you are not using them in the `regress` workflow, it should not make a difference. I see no harm in keeping them and no harm in removing them. You choose.